.github/workflows/py-checkstyle.yml

```diff
@@ -15,7 +15,7 @@ name: Python Checkstyle
 # access to secrets
 on:
   pull_request_target:
-    branches: [ main ]
+    branches: [ main, '0.[0-9]+.[0-9]+' ]
```

Comment:
One quick enhancement suggestion here, let's follow a standard yaml pattern with below snippet everywhere in CI files!
```
branches:
      - main
      - '0.[0-9]+.[0-9]+'
```

Reply:
followed a standard yaml pattern